### ISSUE TITLE
Can't reproduce the MVP performance on FrankaPick 

### ISSUE THREAD

**Issue by Alxead** [ir413/mvp#4]
Hi, thanks for the great project!

I can reproduce the performance of the oracle state model, which shows good stability. 
However, when I try to reimplement the MVP results on FrankaPick, I have some difficulties.
I use the pre-trained MAE you provided. I use 5 seeds: 333, 444, 555, 666, 777.
It seems that IsaacGym **Preview 3** can only be downloaded from the official website now.
I use wandb to log the results. The success rates are shown in the figure below. The success rate has been zero in two out of five runs.

![mvp_FrankaPick](https://user-images.githubusercontent.com/36184713/162114502-b484076a-dba9-40f2-b3fb-a8bd52a6f394.jpg)

The final mean success rate is around 53, shown below, which is far behind the performance (about 85) reported in the paper.

![mvp_FrankaPick_mean](https://user-images.githubusercontent.com/36184713/162115186-d7f5a1a1-c061-4e01-8d0b-98ea87553ffa.jpg)

How can I obtain the results reported in the paper？Is it normal that the success rate on the task is often 0? It seems that MVP exhibits good stability across different seeds (low variance, Figure 5 in the paper), what seeds do you use?

**Comment by mohitsharma0690** [comment]
Hi, I have had similar issues with FrankaPickPixels task, I tried 3 seeds (91, 1729, 625) and 2 of them failed while one got 10% success rate (after 2000 epochs). I am also using "IsaacGym Preview 2" as suggested in the Readme. On the other hand, the performance on FrankaReachPixels seems to be good --- most of the seeds worked in that setting. 

Also, similarly, FrankaPickObjectPixels also did not work for my case (which is clearly harder than FrankaReachPixels). 

Is there some specific hyperparameter that we should be tuning? I tried a few different learning rates, but to no avail.

**Comment by ir413** [comment]
Hi @Alxead and @mohitsharma0690, thanks for trying out the code and taking the time to share your results!

All of the experiments in the paper were run with Isaac Gym Preview 2 and for `seed=0,1,2,3,4`. A few other groups have told us that they reproduced the results with Isaac Gym Preview 3 as well.

As an additional sanity check, following your post we created a fresh environment with Isaac Gym Preview 3 and re-run `FrankaPickPixels` (env [here](https://gist.github.com/ir413/cf3885a826dc332e0f9d4cd8b29ae212)). The resulting curves seem consistent with the ones from the paper. Please see below:

![Screenshot from 2022-04-07 13-34-45](https://user-images.githubusercontent.com/6415258/162299636-43b2402a-5ec1-453c-93f2-bc9cd038fd1d.png)
 
All of these runs use the default config from this repo:

```
python tools/train.py task=FrankaPickPixels logdir=exps/seed-N train.seed=N
```

Hope this helps. Please let us know if you have any further questions!

**Comment by Alxead** [comment]
Hi@ @ir413 
I create a new conda environment follow your configuration and try your seed: 0, 1, 2, 3, 4.  In one of the five runs, the success rate was zero.
If you try more seeds, do you get 0 success rate?
Do you use the same seeds (0, 1, 2, 3, 4) for all the experiments in the paper?

**Comment by ir413** [comment]
Hi @Alxead, I suspect there might be some nondeterminism due to machine and system setup even when the seed is fixed. Following your post, we run the remaining seeds through `seed=10`. The curves seem consistent. Please see below:

![Screenshot from 2022-04-11 00-53-21](https://user-images.githubusercontent.com/6415258/162690400-cc19efd6-6344-4cba-9c1a-b8c13b873c47.png)

We use the same seeds (`0,1,2,3,4`) for all of the experiments in the paper. In addition, we sweep the learning rate and perform the same number of runs for each method in the paper (see Section 4 for more details).

**Comment by ir413** [comment]
Closing this since the original issue seems resolved. @jasonseu, @hekj: Sorry for the late response. Your questions seem different from the original issue, could you please open new issues? (to keep discussions focused; I will respond there)
